src/ActionList/Item.tsx

```diff
@@ -122,6 +136,7 @@ const StyledItem = styled.div<{variant: ItemProps['variant']; item?: ItemInput}
   display: flex;
   border-radius: ${get('radii.2')};
   color: ${({variant, item}) => getItemVariant(variant, item?.disabled).color};
+  position: relative;
```

Comment:
I don't think this one is required since the `:before` moves itself `relative`.

Reply:
Yup, good catch, thanks! Removed in https://github.com/primer/components/pull/1223/commits/2940816381af41209ecca7fe5ab2ca8d1c920807